### ISSUE TITLE
Add BCH support

### ISSUE THREAD

**Issue by graciborski** [airgap-it/airgap-coin-lib#21]
It should be fairly easy as it is almost the same as Bitcoin BTC

**Comment by AndreasGassmann** [comment]
Yes, integrating BCH would probably be very straight forward.

The reason we are hesitant to include bitcoin derivatives is because it opens a new attack surface where users can be tricked into signing a valid transaction that is masked as another currency. In this example, a BTC transaction could be displayed as a BCH transaction.

A vulnerability in many HW wallets was discovered a while ago that exploited this. It looks like BCH is not affected by this, but we'll still have to do some research to make sure we don't miss anything: https://donjon.ledger.com/lsb/014/

Nonetheless, we will discuss this internally.

**Comment by graciborski** [comment]
Yeah, better to be safe than sorry. Thanks for such a quick response!

**Comment by AndreasGassmann** [comment]
You're welcome. I will update this ticket when we have news.

**Comment by graciborski** [comment]
Hey there, any decisions made? :)
